tests/conftest.py

```diff
@@ -132,13 +145,16 @@
 
 @pytest.fixture
 def mbz_builder():
-    def _builder(tmp_path, activities, questionbank_questions=[]):
+    def _builder(tmp_path, activities, sections=[], questionbank_questions=[]):
```

Comment:
Instead of `sections=[]`, what if we define:

```python
DEFAULT_SECTION = {"id": "DEFAULT", "title": "Default Section"}
```

and make this `sections=[DEFAULT_SECTION]`. Then in all of the activities you can do `section_id=DEFAULT_SECTION["id"]` so things are always at least setup properly in the XML.